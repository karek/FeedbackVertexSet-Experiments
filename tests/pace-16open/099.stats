37,62,10
