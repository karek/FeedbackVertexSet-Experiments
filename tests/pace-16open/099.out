8
26 18 22 20 21 24 15 25 
